deploy/windows/px-csi-node-win-shared.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: px-csi-node-win-shared
```

Comment:
Why name the daemonset as `shared`? Let's rename this back to `px-csi-node-win`.
The previous comment was for the storageclass because it was a shared storageclass.